### ISSUE TITLE
More trim for printing.

### ISSUE THREAD

**Issue by bgeorge77** [jemmybutton/byrne-euclid#68]
I just tried to upload the pdf (with lettrines and larger diagrams and line numbers) to Lulu for printing for my class, but it says the trim is too thin. Is there a way to increase the margins?

**Comment by jemmybutton** [comment]
@bgeorge77 I'm sorry for being so slow to respond. This is not difficult in general, but I need to update the version with line numbers, since there were quite a few updates in the code elsewhere. I'm working on it right now. Also, to increase the margins I need to know what is the target paper size. Currently it's made to fit within A5 format (148 by 210 mm), so the easiest thing is to simply make it slightly bigger (say, 158 by 220 mm), but I'm not sure if it's a convenient format for Lulu to print on.

**Comment by bgeorge77** [comment]
I did eventually get the printing going, it looks really nice, I use it all the time in class: 
![PXL_20230927_171141617](https://github.com/jemmybutton/byrne-euclid/assets/8206142/0ef1d8fd-d9d9-4eae-ae6d-6215a25b0dc0)
![PXL_20230927_171212739](https://github.com/jemmybutton/byrne-euclid/assets/8206142/33dca192-2dd6-49f0-9b33-63048c9cfe0b)
![PXL_20230927_171246715](https://github.com/jemmybutton/byrne-euclid/assets/8206142/b9d1539b-8193-48df-83fe-78908c651625)

**Comment by jemmybutton** [comment]
@bgeorge77 It's awesome to see the thing on real paper! Looks really nice in print. I'm really sorry for not making the version I promised yet, it's been a somewhat tough year. At the moment I'm slowly working on a LaTeX version of the book, which should be a tad easier to handle than the current ConTeXt one.
Does it work well in class? Do students find the pictures helpful?

**Comment by bgeorge77** [comment]
Yes! I print out pages from the PDF for the students. The combo of colors AND letters AND line numbers is just so essential, the three keys to unlocking Euclid for young (and old!) students. 
--The big chunky colors grab the eye and differentiate the otherwise samey-samey drawings. I find it VERY hard to run a student through traditional black-and-white Euclid. (I personally also find it difficult to track.)
--The letters make it possible to SPEAK clearly about the drawing--without the letters you're saying, "The blue one, no, the other part of the blue one, the dotty part..." etc. 
--The line numbers help you direct the students' eyes to the proper part of the proof, very essential to make sure they're following. 

I've been using this now for several years, it has been wonderful. 

My only wish? A version of Heath's text, even just the first ut Byrnified.

**Comment by jemmybutton** [comment]
@bgeorge77 Sounds very interesting! I always suspected that colors+letters should be helpful, planned to perform some tests to, maybe, write some sort of research paper so that there's not just tools to design geometry proofs like this, but also some data on how exactly, if at all, this approach is helpful. Did you consider performing a research like this? I could help with designing proofs/theorems for testing and analyzing data, if needed.

Since Heath's text seems to be in public domain, it's technically possible to "byrnify" it, it's just somewhat difficult to support multiple books at once. I also though of "byrnifying" a more modern geometry textbook, but not being a teacher myself, I just couldn't figure out which one makes more sense for the modern-day geometry education.

**Comment by jemmybutton** [comment]
@bgeorge77 A friend of mine, @MarcinCiura, is currently working on a deeply reworked edition of "The Elements," it features letter designations and line numbers, which you found useful. It is in Polish, but we plan to make an English translation. Could you help us with the English version and provide some advice, based on your teaching experience, on how we could improve the book?

**Comment by bgeorge77** [comment]
Absolutely!! That sounds great.

**Comment by bgeorge77** [comment]
Oh, and another book that might need Byrnification is Euclid's "Optics", it it public domain and very ignored, but it is an introduction to perspective geometry. Also pretty short. https://philomatica.org/wp-content/uploads/2013/01/Optics-of-Euclid.pdf

**Comment by MarcinCiura** [comment]
@bgeorge77 May I ask you for your email? You can write it in the comment and then delete it.

**Comment by bgeorge77** [comment]
bgeorge77 at gmail
